### DISCUSSION TITLE
Default visibility of a new View only for the user creating it

### DISCUSSION THREAD

**Discussion by BoneheadFraggle** [home-assistant/frontend#7503] [Dashboard]
I think most admins test and evaluate new views before releasing them to other users, so it would make sense to have only have visibility for the creator as default rather than today's "Everyone".

Would be extra nice feature when partnered with Select/deselect All in View / Visibility (https://github.com/home-assistant/frontend/discussions/7501)

**Comment by matthiasdebaat** [comment]
It depends on the usage situation. My assumption is that it is most common to set visibility for everyone. So 'opt out' would still be the default.

**Reply by BoneheadFraggle** [reply]
That's fine. Maybe a "deselect all" then?
I have about 20 users and a lot of my views are only for a specific user (wallpanel/dashboards). So a "deselect all" would be awesome :-)

**Reply by matthiasdebaat** [reply]
Yeah, I like this option: https://github.com/home-assistant/frontend/discussions/7501
